helm/kubernetes-cluster-prep/bin/get-conjur-cert.sh

```diff
@@ -181,9 +190,9 @@ function ensure_openssl_pod_created() {
     # Create a test deployment if it hasn't been created already
     openssl_pod="$(get_openssl_pod $openssl_deployment)"
     if [ -z "$openssl_pod" ]; then
-        kubectl run "$openssl_deployment" \
+        kubectl create deployment "$openssl_deployment" \
            --image cyberark/conjur-cli:5 \
-           --command sleep infinity
+           -- command sleep infinity
```

Comment:
Can you double-check this, it looks like `command` might not be needed here? I could be wrong.

Reply:
yes, it's not needed. I removed it.